pkg/workceptor/kubernetes.go

```diff
@@ -68,9 +68,8 @@ var ErrPodCompleted = fmt.Errorf("pod ran to completion")
 // ErrImagePullBackOff is returned when the image for the container in the Pod cannot be pulled.
 var ErrImagePullBackOff = fmt.Errorf("container failed to start")
 
-
 // podRunningAndReady is a completion criterion for pod ready to be attached to.
-func podRunningAndReady() func(event watch.Event) (bool, error){
+func podRunningAndReady() func(event watch.Event) (bool, error) {
```

Comment:
this is all changes from linter

Reply:
It would be best to have stuff like this live in a separate commit, to make it easier to figure out what's going on when looking at blames in the future.